.github/ISSUE_TEMPLATE/releases/release_checklist.md

```diff
@@ -12,45 +12,56 @@
-       - [ ] arm64 - `Build Tarball Debian9-Offline_Artifacts/dotnet-smoke-test-prereqs.6.0.xxx.tar.gz`
-     - [ ] Retrieve additional packages from internal MSFT feed using [this project](https://gist.github.com/lbussell/47a3953686c218ede865e305478df74a).
-     - [ ] Create a new tarball of unique packages using [this script](https://gist.github.com/lbussell/5789974491e3d3ed737aac0e8b97b594).
-     - [ ] Upload smoke-test-prereqs tarball to dotnetclimsrc storage account, following the pattern of previous releases for directory and filename.
-         - Never overwrite a tarball. At least change the blob storage virtual dir to represent a new build. This can help avoid timing issues and make it more obvious if stale links were accidentally re-sent rather than new ones.
-1. - [ ] Update dotnet-security-partners
-     - [ ] Submit a PR to the appropriate `release/*` branch on [`dnceng/security-partners-dotnet`](https://dev.azure.com/dnceng/internal/_git/security-partners-dotnet). A branch was automatically created with the source-build-pre-release pipeline, you just have to submit the PR. ([automation tracking issue for this step](https://github.com/dotnet/source-build/issues/3069))
-     - [ ] squash merge the PR once CI finishes successfully.
+1. - [ ] Run the [source-build-pre-release](https://dev.azure.com/dnceng/internal/_build?definitionId=1188) pipeline.
+    - ⚠️ 6.0 / 7.0: [source-build-pre-release](https://dev.azure.com/dnceng/internal/_build?definitionId=1188)
+    - ⚠️ 8.0: [source-build-pre-release-8.0](https://dev.azure.com/dnceng/internal/_build?definitionId=1227)
+    - When staging the pipeline run, click "Resources" and select the final run of Stage-DotNet mentioned above.
+    - This pipeline automatically pushes a branch with the sources that are being released to [`dnceng/security-partners-dotnet`](https://dev.azure.com/dnceng/internal/_git/security-partners-dotnet).
+    - ⚠️ 6.0 / 7.0: This pipeline also uploads the dotnet source tarball to `dotnetclimsrc` with the tarball contents.
+    - The `dotnet/installer` commit that represents the release will be in the logs for "Read Release Info"
+    - The `Get Associated Pipeline Run IDs` build step will contain links to pipelines associated with this release:
+        - ⚠️ 6.0 / 7.0: [dotnet-installer-official-ci](https://dev.azure.com/dnceng/internal/_build?definitionId=286) and [dotnet-installer-source-build-tarball-build](https://dev.azure.com/dnceng/internal/_build?definitionId=1011)
+        - ⚠️ 8.0: [dotnet-dotnet](https://dev.azure.com/dnceng/internal/_build?definitionId=1219)
+        - ⚠️ 8.0: The `dotnet/dotnet` commit that represents the release
+    - [ ] Ensure the `PoisonTests` and `SdkContentTests` are passing.  Warnings indicate a baseline diff and should be inspected carefully.
+1. - [ ] Gather smoke-test prereqs ([🔁 automation tracking issue](https://github.com/dotnet/source-build/issues/3068))
```

Comment:
The formatting changes here broke the nesting.

Here is the rendered MD prior to this change:

<img width="792" alt="image" src="https://user-images.githubusercontent.com/8290530/217578363-3a1ab852-b6b3-4b59-ad34-b9a54bf81771.png">

With this change, notice how the checkboxes are no longer nested under the numbered items.  The nesting is important because the checkbox next to the number indicates that step is completed.  When the checkboxes are not nested, this indication is not so clear.

<img width="803" alt="image" src="https://user-images.githubusercontent.com/8290530/217578695-585ea36f-64b5-482e-82cf-3912f0a18bc4.png">

Reply:
I still see all the checkboxes at the same level.

Reply:
Ah, my bad. Fixed

Reply:
Ah, my bad. Fixed

Reply:
This should apply to all sections with sub-checkboxes.  There are two others.  Because of that, I think you will notice the normal bulleted items were double nested for consistency .

Reply:
Will do

Reply:
Ah, the indentation needs to be 5 spaces..